releng/releng-target/lsp4j.target.target

```diff
@@ -8,8 +8,8 @@
 			<unit id="com.google.gson.source" version="2.8.2.v20180104-1110"/>
 			<unit id="com.google.guava" version="27.1.0.v20190517-1946"/>
 			<unit id="com.google.guava.source" version="27.1.0.v20190517-1946"/>
-			<unit id="javax.websocket" version="0.0.0"/>
-			<unit id="javax.websocket.source" version="0.0.0"/>
+			<unit id="jakarta.websocket" version="0.0.0"/>
```

Comment:
interestingly i dont find this in the list of orbit
https://download.eclipse.org/tools/orbit/downloads/drops/R20191126223242/

Reply:
not yet available: https://git.eclipse.org/r/c/orbit/orbit-recipes/+/174146

Reply:
i assume the update site needs to be changed to
2021-03 along with
gson once both the new gson and websocket appear there
https://github.com/eclipse/lsp4j/compare/cd_issue488

Reply:
I think we can modify the update site to 2021-03 with gson in a separated PR. Then I will rebase this PR on it. There might need to modify again to point to some Integration update site in this PR but that won't be too much of a problem.

Reply:
yes i am gonna open the gson PR once the M1 of orbit is available and thus the new version appears

Reply:
gson and jakarta.websocket are now available